config/routes.rb

```diff
@@ -3,4 +3,9 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
+  get "/posts/new", to: "posts#new"
+  post "posts/new", to: "posts#create"
```

Comment:
- [ ] Please modify your routes to take into account that a new post belongs to a user. An easy way would be to include the new and create actions in the resource routes you which you already have.